lib/pure/prelude.nim

```diff
@@ -10,11 +10,13 @@
 when defined(nimdoc) and isMainModule:
   from std/compileSettings import nil
   when compileSettings.querySetting(compileSettings.SingleValueSetting.projectFull) == currentSourcePath:
+    ## This module is deprecated. Import the [batteries] module instead.
+    ##
     ## This is an include file that simply imports common modules for your convenience.
     runnableExamples:
       include std/prelude
         # same as:
-        # import std/[os, strutils, times, parseutils, hashes, tables, sets, sequtils, parseopt]
+        # import std/[os, strutils, times, parseutils, hashes, tables, sets, sequtils, parseopt, strformat]
```

Comment:
I did not know that strformat is being deprecated. That is actually quite surprising. Wasn't it (relatively) recently introduced? What will replace it?

Also, I think that the batteries module should be a strict superset of prelude, which (at this point) still includes it...

Reply:
strformat is stable and will **not** be deprecated, I don't know where this idea comes from.

Reply:
I think they mean prelude is deprecated.

Reply:
Yes, I meant that since we are deprecating prelude, we might not want to change it by adding strformat to it. I guess it might have been a change in prelude left by mistake.

Reply:
Ah, sorry, I misunderstood you @pietroppeter. You are right, that was not meant to be part of the changes, sorry.